src/components/WeeklySummariesReport/WeeklySummariesReport.css

```diff
@@ -11,6 +11,17 @@
   border: 1px solid transparent;
   border-color: #fff #dee2e6 #dee2e6;
 }
+body.dark .tab-content {
+  background-color: #333;
+  border: 1px solid transparent;
+  border-color: #fff ;
+}
+body.dark .team-member-tasks{
+  background-color: #333;
+}
+body.dark .header-box, body.dark .header-box h1{
+  color: white;
```

Comment:
The weekly reports page in dark mode is too light due to the use of white as a text or background color
<img width="1440" alt="Screenshot 2023-10-26 at 12 05 27 PM" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/61841929/5be9f3ac-09a6-4279-ba5e-a2ddc970d26a">

Reply:
Everything else in the dark mode looks good

Uploading Screen Recording 2023-10-26 at 10.57.14 AM.mov…